lib/reporter.js

```diff
@@ -24,7 +24,7 @@ class JunitReporter extends events.EventEmitter {
             for (let cid of Object.keys(this.baseReporter.stats.runners)) {
                 const capabilities = this.baseReporter.stats.runners[cid]
                 const xml = this.prepareXml(capabilities)
-                this.write(capabilities, cid, xml)
+                this.write(capabilities, cid, xml, this.config)
```

Comment:
No need to pass `this.config` in as parameter. It will be accessible in `write` via `this.config` too.